### PR TITLE
Replace deprecated slash for division

### PR DESCRIPTION
**Pull request by kimoshinowo98**
Ticket: https://webjobs.agepartnership.co.uk/desk/tickets/5687300/messages

Changes:
- Use sass-migrator to change division slashes to multiplications or to use math.div()

Checks:
- Non-malicious
- Sense

### CODE REVIEW

File: base/_grid.scss [hunk 1]
```diff
@@ -35,9 +35,9 @@
 // Example use: @include grid-cell(2,2,1,1)
```
Comment (ryanfitton-ap) [inline base/_grid.scss:35]: We may need to add `@use 'sass:list';` to the top of the file, unless this is already added by Gulp when we compile?

I have tested with a test file: `test.scss`:
```
@use 'sass:list';

@mixin grid-cell($col-start, $col-end, $row-start, $row-end) {
    -ms-grid-column: $col-start;
    grid-column: list.slash($col-start, $col-end);
    -ms-grid-column-span: $col-end - $col-start;
    -ms-grid-row: $row-start;
    grid-row: list.slash($row-start, $row-end);
    -ms-grid-row-span: $row-end - $row-start;
}

.test {
    @include grid-cell(2, 2, 1, 1)
}
```

And running:  `sass test.scss test.css`, i'm running Sass `1.57.1 compiled with dart2js 2.18.6`

**Review by ryanfitton-ap — CHANGES_REQUESTED** [review]
We may need to add `@use 'sass:list';` to the top of the file, unless this is already added by Gulp when we compile?

I have tested with a test file: `test.scss`:
```
@use 'sass:list';

@mixin grid-cell($col-start, $col-end, $row-start, $row-end) {
    -ms-grid-column: $col-start;
    grid-column: list.slash($col-start, $col-end);
    -ms-grid-column-span: $col-end - $col-start;
    -ms-grid-row: $row-start;
    grid-row: list.slash($row-start, $row-end);
    -ms-grid-row-span: $row-end - $row-start;
}

.test {
    @include grid-cell(2, 2, 1, 1)
}
```

And running:  `sass test.scss test.css`, i'm running Sass `1.57.1 compiled with dart2js 2.18.6`

**Comment by ryanfitton-ap** [comment]
Like you have done in this other PR https://github.com/AgePartnership/web.agepartnership.co.uk/blob/642991ac3006cfaa75e002236ee3ddafbb87298e/resources/css/landing/equity-release/202206-search-journey/_content-highlight-banner.scss with `@use 'sass:math';`

**Comment by kimoshinowo98** [comment]
Tested on https://github.com/AgePartnership/web.agepartnership.co.uk/pull/3353 and seems to work

**Review by ryanfitton-ap — APPROVED** [review]
If working happy to improve. If we see any issues in the future can be referred back to this PR noting the fix is to add `@use 'sass:list';`

**Comment by beccadawson** [comment]
> If working happy to approve. If we see any issues in the future can be referred back to this PR noting the fix is to add `@use 'sass:list';`

Am confused, we have this in the file already?

**Pull request merged**